pages/deposit.tsx

```diff
@@ -416,45 +427,79 @@ const DepositPage: NextPage = () => {
             ]}
           />
         )}
-        {method === "buy" && currency === "dot" && paymentMethod === "card" && (
-          <div className={"grid gap-3 " + `grid-cols-1`}>
-            <ResultButtons
-              items={[{ label: "Banxa *", url: "https://banxa.com/" }]}
-            />
-            <div className="mt-2">
-              * — Complete purchase on the Banxa page then return to this page
-              and select the Deposit tab to continue
+        {method === "buy" &&
+          currency === "dot" &&
+          paymentMethod === "card" &&
+          encodedAddress && (
+            <div className={"grid gap-3 " + `grid-cols-1`}>
+              <ResultButtons
+                items={[
+                  {
+                    label: "Banxa",
+                    url: `https://checkout.banxa.com/?fiatAmount=50&fiatType=EUR&coinAmount=8&coinType=DOT&lockFiat=false&orderMode=BUY&walletAddress=${encodedAddress}`,
+                  },
+                ]}
+              />
+              <div className="flex items-center gap-2 mt-7">
+                <Image
+                  src="/currencies/dot.png"
+                  width={25}
+                  height={25}
+                  alt="Polkadot currency"
+                />
+                <div>Polkadot Address:</div>
+                <div className="flex font-semibold">
```

Comment:
a word break is required here to prevent layout shift

![Screenshot 2023-09-05 at 11 06 31 PM](https://github.com/zeitgeistpm/ui/assets/14626415/c6c9651e-7347-49db-bd9d-6a0b3ab53334)